packages/contract/lib/utils/index.js

```diff
@@ -133,22 +133,27 @@ const Utils = {
   },
 
   // Extracts optional tx params from a list of fn arguments
-  getTxParams(methodABI, args) {
+  getTxParams(methodABI, args, ignoreDefaultGasPriceParams = false) {
```

Comment:
Might it be worth making a comment describing why this last param might be useful? Specifically about why we implemented it?

Reply:
Sure, that makes sense.  I've added a comment.